pkg/config/config_template.yaml

```diff
@@ -1107,6 +1107,19 @@ api_key:
   #   process: 10
   #   process_realtime: 2
 
+  ## @param process_discovery - custom object - optional
+  ## Specifies custom settings for the `process_discovery` object.
+  # process_discovery
+      ## @param enabled - boolean - optional - default: enabled
+      ## Toggles the `process_discovery` check. This check gathers information about possible integrations
+      ## for the user to enable.
+      # enabled: true
+
+      ## @param interval - duration - optional - default: 4h
+      ## An interval in hours that specifies how often the process discovery check should run.
+      # interval: 4h
```

Comment:
should we inform the interval as `4h` or `4`?

Reply:
According to the RFC it should be 4h since we're reading in a duration.

https://github.com/DataDog/architecture/blob/master/rfcs/process-discovery-for-onboarding/rfc.md#process-agent